packages/server/src/lib/email.js

```diff
@@ -123,11 +186,74 @@ async function sendGrantAssignedEmail({ grantId, agencyIds, userId }) {
     agencies.forEach((agency) => module.exports.sendGrantAssignedNotficationForAgency(agency, grantDetail, userId));
 }
 
+async function sendGrantDigestForAgency(agency) {
+    const newGrants = await db.getNewGrantsForAgency(agency);
+
+    if (newGrants.length === 0) {
+        return undefined;
+    }
+
+    const grantDetails = [];
+    newGrants.slice(0, 3).forEach((grant) => grantDetails.push(module.exports.getGrantDetail(grant)));
```

Comment:
Check with knex if it can get total count while getting the first 3 results only

Reply:
make some screenshots of metrics dashboard in render to get an idea for typical usage and memory usage.